[libpq]: Link error with static openssl

When building libpq with static openssl it fails at the linker step with the following:

```
Microsoft (R) Incremental Linker Version 14.16.27030.1
Copyright (C) Microsoft Corporation.  All rights reserved.

   Creating library libpqd.lib and object libpqd.exp
libeay32.lib(e_capi.obj) : error LNK2019: unresolved external symbol __imp_CertOpenStore referenced in function capi_open_store
libeay32.lib(e_capi.obj) : error LNK2019: unresolved external symbol __imp_CertCloseStore referenced in function capi_find_key
libeay32.lib(e_capi.obj) : error LNK2019: unresolved external symbol __imp_CertEnumCertificatesInStore referenced in function capi_find_cert
libeay32.lib(e_capi.obj) : error LNK2019: unresolved external symbol __imp_CertFindCertificateInStore referenced in function capi_find_cert
libeay32.lib(e_capi.obj) : error LNK2019: unresolved external symbol __imp_CertDuplicateCertificateContext referenced in function capi_load_ssl_client_cert
libeay32.lib(e_capi.obj) : error LNK2019: unresolved external symbol __imp_CertFreeCertificateContext referenced in function capi_find_key
libeay32.lib(e_capi.obj) : error LNK2019: unresolved external symbol __imp_CertGetCertificateContextProperty referenced in function capi_cert_get_fname
libpqd.dll : fatal error LNK1120: 7 unresolved externals
```
Per the [OpenSSL notes for windows](https://github.com/openssl/openssl/pull/1063), when linking openssl static libraries, `crypt32` should be part of the libraries to be linked as well.

PR to fix this to follow

--- a/ports/libpq/CMakeLists.txt
+++ b/ports/libpq/CMakeLists.txt
@@ -141,7 +141,7 @@ endif()
 
 target_link_libraries(libpq PRIVATE OpenSSL::SSL)
 if(WIN32)
-    target_link_libraries(libpq PRIVATE ws2_32 secur32 advapi32 shell32)
+    target_link_libraries(libpq PRIVATE ws2_32 secur32 advapi32 shell32 crypt32)
 endif()
 target_include_directories(libpq PRIVATE src/include src/port ${CMAKE_CURRENT_BINARY_DIR}/include)
 set_target_properties(libpq PROPERTIES WINDOWS_EXPORT_ALL_SYMBOLS ON)

--- a/ports/libpq/CONTROL
+++ b/ports/libpq/CONTROL
@@ -1,4 +1,4 @@
 Source: libpq
-Version: 9.6.1-6
+Version: 9.6.1-7
 Description: The official database access API of postgresql
 Build-Depends: openssl, zlib (linux)

